src/codelens/TestCodeLensProvider.ts

```diff
@@ -139,7 +138,7 @@ export class TestCodeLensProvider implements CodeLensProvider {
             return '?';
```

Comment:
Does vscode has an icon to represent the unknown state?

Reply:
A related one might be:

![image](https://user-images.githubusercontent.com/6193897/67266314-81dc5e80-f4e2-11e9-8960-50ee0bc68036.png)

But looks too complicated.

Or maybe we can just show nothing if the state is unknown.

Reply:
$(question)

Reply:
Per offline discussion. We will leave the question icon as what it is.